### PR TITLE
chore: Clean up storage docs

### PR DESCRIPTION
**Pull request by jimblanc**
#### Description of changes:
Cleans up storage docs based on feedback on the main PR. Replaced all tabs with 2 spaces as per the style guide.

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [x] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [x] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/fragments/lib-v1/storage/js/copy.mdx [hunk 1]
```diff
@@ -38,13 +38,14 @@ You can copy a file from the specified key to another key within the same [File
 ```javascript
 import { copy, list } from 'aws-amplify/storage';
 
-console.log(await list({ prefix : 'copied/' })); // []
```
Comment (jimblanc) [inline src/fragments/lib-v1/storage/js/copy.mdx:41]: Removed these preceding console logs as I don't think they were adding much, can add back if people feel strongly.

File: src/fragments/lib-v1/storage/js/download.mdx [hunk 2]
```diff
@@ -125,45 +126,18 @@ import { downloadData } from 'aws-amplify/storage';
...
 }
 ```
 
 <Callout>
-To get the metadata in result for all read apis we have to configure user defined metadata in CORS
-
-Update your bucket's CORS Policy to look like:
+To get the metadata in result for all APIs you have to configure user defined metadata in CORS.
 
-```json
```
Comment (jimblanc) [inline src/fragments/lib-v1/storage/js/download.mdx:141]: Replaced example with just a link to the setup instructions in the getting started guide.

**Review by AllanZhengYP — APPROVED** [review]
Thanks @jimblanc for the fast followup!

**Review by kvramyasri7 — APPROVED** [review]
Thanks for doing this Jim👍

**Review by ashwinkumar6 — APPROVED** [review]
Thanks a lot for updating this @jimblanc !!

File: src/fragments/lib-v1/storage/js/getting-started.mdx [hunk 3]
```diff
@@ -68,6 +68,11 @@ Amplify.configure({
     S3: {
       bucket: 'XXX', //REQUIRED -  Amazon S3 bucket name
       region: 'XX-XXXX-X', //OPTIONAL -  Amazon service region
+    }
+  }
+}, {
+  Storage: {
+    S3: {
```
Comment (kvramyasri7) [inline src/fragments/lib-v1/storage/js/getting-started.mdx:75]: Should `isObjectLockEnabled: true` be in above `Storage` block why do we have 2 separate S3 configs here?

Comment (jimblanc) [reply src/fragments/lib-v1/storage/js/getting-started.mdx:75]: This is correct, object lock enabled status is toggled with Library options, which is the 2nd param to configure

**Pull request merged**